week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -13,9 +13,19 @@
  *      .....
  * </div>
  */
-function exerciseOne(arrayOfPeople) {
-  let content = document.querySelector("#content");
-}
+ function exerciseOne(arrayOfPeople) {
+   let content = document.querySelector("#content");
+   for(i = 0; i < arrayOfPeople.length; i++){
+     let person  = arrayOfPeople[i];
+     let name = document.createElement("h1");
+     name.textContent = person.name;
+     let job = document.createElement("h2");
+     job.textContent = person.job;
+     content.appendChild(name)
+     content.appendChild(job)
+ console.log(name)
```

Comment:
A tiny formatting error here. Could you use something like this?
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode